build.sbt

```diff
@@ -12,17 +12,17 @@ val zkDependency = "org.apache.zookeeper" % "zookeeper" % zkVersion excludeAll(
   ExclusionRule("javax.jms", "jms")
 )
 val slf4jVersion = "1.7.21"
-val jacksonVersion = "2.9.8"
+val jacksonVersion = "2.9.9"
```

Comment:
There is no `2.9.8` for 2.13.0, so we have to bump it independently, we have to bump it before and rebase this on top of that.

I have a branch where that's done --https://github.com/twitter/util/compare/develop...martijnhoekstra:pre213 is a diff of a reasonable bump of dependencies and elimination of deprecations that go along with that. But to be honest, I'm not sure what value that brings, other than better baseline for benchmarking maybe? I doubt it would matter much.